options.go

```diff
@@ -0,0 +1,66 @@
+package cobra2snooty
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+const optionsHeader = `.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+`
+
+var ErrMissingDescription = errors.New("missing description")
+
+func printArgs(buf *bytes.Buffer, cmd *cobra.Command) error {
+	if args, ok := cmd.Annotations["args"]; ok {
+		buf.WriteString("Arguments\n")
+		buf.WriteString("---------\n\n")
+		buf.WriteString(optionsHeader)
+		var requiredSlice []string
+		if requiredArgs, hasRequired := cmd.Annotations["requiredArgs"]; hasRequired {
+			requiredSlice = strings.Split(requiredArgs, ",")
+		}
+
+		for _, arg := range strings.Split(args, ",") {
+			required := stringInSlice(requiredSlice, arg)
+			if description, hasDescription := cmd.Annotations[arg+"Desc"]; hasDescription {
+				line := fmt.Sprintf("   * - %s\n     - string\n     - %v\n     - %s", arg, required, description)
+				buf.WriteString(line)
+			} else {
+				return fmt.Errorf("%w: %s", ErrMissingDescription, arg)
```

Comment:
if we can't find a description error the doc generation

Reply:
💯

Reply:
cc @andreaangiolillo